include/megaapi_impl.h

```diff
@@ -2924,6 +2924,10 @@ class MegaApiImpl : public MegaApp
         bool sync_syncable(Sync *, const char*, string *, Node *) override;
         bool sync_syncable(Sync *, const char*, string *) override;
         void syncupdate_local_lockretry(bool) override;
+
+        // for the exclusive use of sync_syncable
+        unique_ptr<FileAccess> mSyncable_fa;
+        std::mutex mSyncable_fa_mutex;
```

Comment:
Normally we don't use snake case according to our guidelines..

Reply:
'fa' is a well established name for FileAccess objects though, and would look odd with a capital F... if someone else wants to adjust it, I won't object :-)